helm/plater/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 image:
   plater:
     repository: renciorg/plater-clustered
-    tag: "1.1.2-1"
+    tag: "1.1.2-2rc"
```

Comment:
are you going to leave this as a release candidate?

Reply:
opps sorry reverting back

Reply:
should be fixed now